src/core/borrower/collect/collector.sol

```diff
@@ -53,11 +53,12 @@ contract Collector is DSNote, Auth {
 
     // --- Data ---
     RegistryLike threshold;
-    struct Lot {
-        address usr;
-        uint    wad;
+
+    struct SellingPrice {
+        address buyer;
+        uint    nftPrice;
     }
-    mapping (uint => Lot) public tags;
+    mapping (uint => SellingPrice) public prices;
```

Comment:
IMO is not a good refactor. Because it's not just prices but also users who may buy.

`SellingPrice` is grammatically not quite correct. It would have to be SalePrice or OfferPrice. A good compromise IMO could be:

Option: an option allows someone to execute a trade.

Reply:
change it to option